kata-webhook/deploy/webhook.yaml

```diff
@@ -24,7 +24,11 @@ spec:
           imagePullPolicy: Always
           env:
             - name: RUNTIME_CLASS
-              value: kata
+              valueFrom:
+                configMapKeyRef:
+                  name: kata-webhook
+                  key: runtime_class
+                  optional: true
```

Comment:
Is it still uses "kata" as default? can it be done?

Reply:
Good point @snir911. I was looking at pod's `env` specification (https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/pod-v1/#environment-variables-1), it is not possible to set a default value. `configMapKeyRef` also doesn't allow a default value. Note that in https://github.com/kata-containers/tests/blob/main/kata-webhook/main.go#L77 it is going to use `kata` if `RUNTIME_CLASS` is not export.

Anyway, I need to test it works fine if the configMap isn't deployed.

Reply:
@snir911 I tested it locally; the change proposed on kata-webhook/deploy/webhook.yaml will not break the default workflow (i.e. when the configMap isn't deployed)